src/main/java/com/conveyal/datatools/editor/controllers/api/FareController.java

```diff
@@ -140,42 +127,51 @@ public static Object updateFare(Request req, Response res) {
                 fare.gtfsFareId = "CAL_" + fare.id.toString();
             }
 
-//            fare.addDerivedInfo(tx);
-
             tx.fares.put(fare.id, fare);
-
-            Object json = Base.toJson(fare, false);
-
             tx.commit();
-
-            return json;
+            return fare;
         } catch (HaltException e) {
             LOG.error("Halt encountered", e);
```

Comment:
No, not you.  I was running into issues where halts were getting caught by the `catch (Exception e) {}` clause.  See this comment: https://github.com/perwendel/spark/blob/e2854295f7c484671ce69bff7ec37fcb0c413a32/src/main/java/spark/Service.java#L562-L564